### PR TITLE
A sixth pull request on Metrics App (Quizzical)

### PR DESCRIPTION
**Pull request by dlwub**
Based on the reviewer's comment:
- Files are renamed to enable easy navigation 
- A stateless filter is added on the Home page
- A back button that returns to the front page is included on the details page and on the side of each question
- Two different colors are applied to the questions (on the details page) alternatively

### CODE REVIEW

File: src/redux/__test__/Questions.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  it('checks if the count of answered questions is zero initially', async () => {
+    expect(store.getState().questions.count).toEqual(0);
+  });
+  it('checks if the number of correctly answered questions is five ', async () => {
+    store.getState().questions.list.forEach((question) => {
+      store.dispatch(setChoice([question.id, question.correctAnswer]));
+    });
+    store.dispatch(countAnswer());
+    expect(store.getState().questions.count).toEqual(5);
+  });
```
Comment (HAMMAS-SALEEM) [inline src/redux/__test__/Questions.test.js:23]: image

- [ ] All your tests are not passing. Kindly make your application pass all the tests :blush:

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default function Header() {
+  return (
+    <div className="header">Quizzical</div>
+  );
```
Comment (HAMMAS-SALEEM) [inline src/components/Header.js:6]: ### Your Header
image
### Design Header
missing-icons (copy)


- [ ] Your header is missin those icons shown in the design image abolve. Kindly add them to make your design look professional :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
Kindly fix your linters

File: src/redux/__test__/Questions.test.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+});
```
Comment (VICTOR-CHUKWU) [inline src/redux/__test__/Questions.test.js:24]: - [ ] Hello @dlwub , some of your tests are failing, kindly fix them
image

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @dlwub , 
## STATUS: CHANGES REQUIRED :recycle: 
Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/redux/__test__/Questions.test.js (hunk 3 above)
Comment (dlwub) [reply src/redux/__test__/Questions.test.js:24]: Hi @ VICTOR-CHUKWU
All tests are passing on my computer. 
Screenshot (43)

**Review by cynthiainga — APPROVED** [review]
Hi @dlwub, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Used correct Github flow ✔️ 
- [ ] All unit tests are passing ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**